specification/ml/preview_datafeed/MlPreviewDatafeedResponse.ts

```diff
@@ -18,7 +18,5 @@
  */
 
 export class Response<TDocument> {
-  body: {
-    data: TDocument[]
-  }
+  body: TDocument[]
```

Comment:
I think you should use `Array<T>` here, the compiler is not smart enough yet to understand the shorthand syntax.

```suggestion
  body: Array<TDocument>
```